### ISSUE TITLE
I am this after i type spyder in Anaconda prompt and spyder is still crashing.

### ISSUE THREAD

**Issue by swapniljn8** [spyder-ide/spyder#9317]
```
Start of MainWindow constructor
appusermodelid: 0
End of MainWindow constructor
*** Start of MainWindow setup ***
  ..core actions
  ..toolbars
  ..tools
  ..sift?
  ..plugin: internal console
    ..internal console: initializing
  ..plugin: working directory
Loading help...
Loading outline explorer...
Loading editor...
Assertion failed: Connection reset by peer [10054] (C:\ci\zeromq_1549025594292\work\src\signaler.cpp:371)
Assertion failed: Software caused connection abort [10053] (C:\ci\zeromq_1549025594292\work\src\signaler.cpp:192)
[17152:13292:0510/133910.517:ERROR:broker_win.cc(59)] Error reading broker pipe: The pipe has been ended. (0x6D)
```

**Comment by ccordoba12** [comment]
This problem is caused by your firewall and/or antivirus. Please deactivate them and try again, that should fix your problem.

**Comment by swapniljn8** [comment]
> This problem is caused by your firewall and/or antivirus. Please deactivate them and try again, that should fix your problem.

No,  I stopped my firewall it is still not running and i am using Quick Total protection as my firewall and my friend too is using the same but is running in his laptop but not in mine.
Spyder is not opening its editor and whole UI, only the main window come on which Spyder3 is written and then after Loading editor.... it closes and nothing happen after that.
I have tried from Spyder shortcut
I have tried from Anaconda
I have tried from Anaconda terminal 
nothing helping.
